README.md

```diff
@@ -8,7 +8,7 @@ Lovely is a lua injector which embeds code into a [LÖVE 2d](https://love2d.org/
 
 1. Download the [latest release](https://github.com/ethangreen-dev/lovely-injector/releases) for Windows. This will be `lovely-x86_64-pc-windows-msvc.zip`.
 2. Open the .zip archive, copy `version.dll` into the game directory. You can navigate to the game's directory by right-clicking the game in Steam, hovering "Manage", and selecting "Browse local files".
-3. Put one or more mods into the mod directory (NOT the same as the game directory). This should be `%AppData%/Balatro/Mods` (if you are modding Balatro).
+3. Put one or more mods into the mod directory (NOT the same as the game directory). This should be `%AppData%/Balatro/Mods`, if you are modding Balatro ([`Home/.steam/steam/steamapps/compatdata/2379780/pfx/drive_c/users/steamuser/AppData/Roaming/Balatro`](https://steamcommunity.com/sharedfiles/filedetails/?id=3178949415) if you are on Steam Deck).
```

Comment:
My only real critique is that `Home/` should be replaced with `~/`, otherwise this looks good. It makes the install guide a little more difficult to read but we should probably improve that by moving it over to a dedicated wiki page.

Reply:
Thanks. I added your changes and also @antipatico's, now in a bulleted list for all the Windows + Proton / Wine platforms. I think this is a more intuitive way to organize the instruction and should be easier to read.

Reply:
One note: why exactly do you think `Home/` should be replaced with `~/`? As you can see in the Steam Deck interface is the guide I linked, the directory is clearly marked as `Home/`, so I think this change will be confusing to users:
![CleanShot 2025-01-13 at 06 33 15@2x](https://github.com/user-attachments/assets/2406cae8-9b01-4a92-8dfd-062bb3b4cff6)

I've changed the path back `Home/` in anticipation that you will agree but I can change it back to `~/` if you insist on the change as contingent for your approval.

Reply:
I also added a note about `~/` being *Home* if you are editing in the GUI.